test/oppfolgingsoppgave/OppfolgingsoppgaveModalTest.tsx

```diff
@@ -104,7 +105,11 @@ describe("Oppfolgingsoppgave", () => {
       });
       userEvent.click(openModalButton);
 
-      expect(await screen.findByText("Velg oppfølgingsgrunn")).to.exist;
+      expect(
```

Comment:
Kan også tilpasse navnet på `it`-en over her til å inkludere tekst. Og kanskje legge til sjekk på tekst-input i den første it-en. Og en egen test for at tekst-input ikke vises når man velger annet.